diff -uNr CSFML-2.1/src/SFML/CMakeLists.txt CSFML-2.1-patched/src/SFML/CMakeLists.txt
--- CSFML-2.1/src/SFML/CMakeLists.txt	2013-03-24 13:14:58.000000000 +0100
+++ CSFML-2.1-patched/src/SFML/CMakeLists.txt	2015-06-28 19:41:46.658771521 +0200
@@ -6,7 +6,7 @@
 include_directories(${CMAKE_SOURCE_DIR}/src)
 
 # define the path of our additional CMake modules
-set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+#set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 # set the output directory for CSFML libraries
 set(LIBRARY_OUTPUT_PATH "${CMAKE_BINARY_DIR}/lib")
